### DISCUSSION TITLE
How to make a node that will not become a candidate?

### DISCUSSION THREAD

**Discussion by LazyMechanic** [databendlabs/openraft#1235] [Q&A] [answered]
I want an “arbiter” node that will help the network reach consensus, but will never be a leader itself.  

Do I understand correctly that setting `enable_elect = false` will help me?
```rust
let cfg = openraft::Config {
    enable_elect: false,
    ..Default::default(),
};
```
Will a node with this setting participate in the voting of the other nodes? 
With a system of 3 nodes, if one node goes out, will the system state still be consistent?

**Comment by drmingdrmer** [comment, marked as answer]
Yes. `enable_elect: false` will disable election for this node.
And yes, this node will still vote for other leader.

Yes, if one nodes goes offline, but not yet being removed from the cluster with `Raft::change_membership()`, one of the other two nodes will elect and become the leader. The cluster is still in a consistent state.

If a node is removed from the cluster with `Raft::change_membership()`, the state is also consistent, but a two nodes cluster won't tolerate any node failure.
